From 66d80a210bf2a6e5daa74632d3746f54ab31095e Mon Sep 17 00:00:00 2001
From: ariel aviad <aaviad@habana.ai>
Date: Mon, 09 Dec 2024 13:22:52 +0200
Subject: [PATCH] [FSW-6867] bin: elf: prep for refact pldm target

duplicate pldm bin files and elf files from pldm
to palad
prepering to change pldm target to palad

Change-Id: Ic5c8bcb292d803fa1c9b84e6a5816fc8859156e1
Signed-off-by: ariel aviad <aaviad@habana.ai>
---

diff --git a/bin/gaudi2_arc/palad/arcboot_rom.bin b/bin/gaudi2_arc/palad/arcboot_rom.bin
new file mode 100644
index 0000000..707165c
--- /dev/null
+++ b/bin/gaudi2_arc/palad/arcboot_rom.bin
Binary files differ
diff --git a/bin/gaudi2_arc/palad/arcboot_rom.elf b/bin/gaudi2_arc/palad/arcboot_rom.elf
new file mode 100644
index 0000000..f51d261
--- /dev/null
+++ b/bin/gaudi2_arc/palad/arcboot_rom.elf
Binary files differ
diff --git a/bin/gaudi3_arc/palad/arcboot_rom.bin b/bin/gaudi3_arc/palad/arcboot_rom.bin
new file mode 100644
index 0000000..eba1646
--- /dev/null
+++ b/bin/gaudi3_arc/palad/arcboot_rom.bin
Binary files differ
diff --git a/bin/gaudi3_arc/palad/arcboot_rom.elf b/bin/gaudi3_arc/palad/arcboot_rom.elf
new file mode 100644
index 0000000..2162103
--- /dev/null
+++ b/bin/gaudi3_arc/palad/arcboot_rom.elf
Binary files differ
